src/lib/components/selectv2/Selectv2.component.tsx

```diff
@@ -235,6 +235,10 @@ function SelectBox({
   className,
   ...rest
 }: SelectProps) {
+  if (defaultValue & value)
+    console.error(
+      'The `defaultValue` will be overridden by the `value` if they are set at the same time.',
+    );
```

Comment:
nit: prefer using a block with condition when using it on multiline
```suggestion
  if (defaultValue && value) {
    console.error(
      'The `defaultValue` will be overridden by the `value` if they are set at the same time.',
    );
  }
```